### ISSUE TITLE
Google Drive 404 when trying to download models

### ISSUE THREAD

**Issue by thedirptastic** [chrisdonahue/LakhNES#16]
When I try to download the models, I get a 404 error

**Comment by alesaccoia** [comment]
Same problem here. Did you find an alternative download location? I don't get how to reproduce the training procedure, too

**Comment by alesaccoia** [comment]
@chrisdonahue could you help?

**Comment by chrisdonahue** [comment]
Hi. Sorry about this, not sure why the Google Drive links expired. I just spent some time looking through old files but I unfortunately cannot find these checkpoints :(

I wonder if someone else might have downloaded them? The README has the checksums for each file so they can at least be validated if found

**Comment by alesaccoia** [comment]
Thank you, let's see if someone comes up.

I have been trying for some days to complete the missing training parts and update the code to the current pytorch apis (https://github.com/alesaccoia/LakhNES) - will send a PR when done - still I am unable to repdroduce the results of Lakh + fine tuning on  Nesdb, perplexity remains quite high. Steps taken:

- Downloaded the full Lakh dataset
- Ran the [adapt_lakh_to_nes.py](https://github.com/alesaccoia/LakhNES/blob/master/data/adapt_lakh_to_nes.py) utility
- Created [this utility](https://github.com/alesaccoia/LakhNES/blob/master/data/convert_midi_folder_to_tx1.py) that converts the midi files to tx1
- Separated into train, valid, test set
- Ran on two Tesla T4 after updating the multi_cpu code that was throwing an error, using the only options I could infer from the paper:

```bash
python3 train.py --data ../data/nesmdb_tx1 --dataset nesmdb --cuda --batch_size 30 --n_head 8 --multi_gpu
```

If you could help reproducing the results then I'd upload the checkpoints, this project is so cool let's revive it!

**Comment by alesaccoia** [comment]
Ah, @chrisdonahue - maybe you've changed permissions but they are still on your g drive?

**Comment by NeuroLord** [comment]
Any updates on the checkpoints?

**Comment by AlexEduardo-zip** [comment]
I'm waiting too
